platform_versioned_docs/version-24.2/troubleshooting_and_faqs/data_studios_troubleshooting.mdx

```diff
@@ -11,6 +11,10 @@ In your interactive analysis environment, open a new terminal and type `ls -la /
 
 ![](./_images/data_studios_notebook_fusion.png)
 
+## Session size limited by compute environment advanced options: Head job CPUs and Head job memory
+
+ When adding a compute environment, setting the Advanced options "Head job CPUs" and “Head job memory” for Nextflow **also applies** to any studio session created in the compute environment. This is because studio sessions are managed by the Nextflow runner job. To avoid artifically constraining the resources of your studio sessions, **do not define these optional compute environment settings**.
```

Comment:
```suggestion
 When adding a compute environment, setting the Advanced options **Head job CPUs** and **Head job memory** for Nextflow **also applies** to any studio session created in the compute environment. This is because studio sessions are managed by the Nextflow runner job. To avoid artifically constraining the resources of your studio sessions, **do not define these optional compute environment settings**.
```